kubectl-openebs/cli/command/get/volume_list.go

```diff
@@ -79,16 +79,30 @@ func RunVolumesList(cmd *cobra.Command, openebsNs string, vols []string) error {
 	// give output according to volume status
 	var rows []metav1.TableRow
 	for _, item := range cvols.Items {
-		rows = append(rows, metav1.TableRow{Cells: []interface{}{
-			item.ObjectMeta.Namespace,
-			item.ObjectMeta.Name,
-			item.Status.Phase,
-			item.VersionDetails.Status.Current,
-			util.ConvertToIBytes(item.Status.Capacity.String()),
-			pvols[item.ObjectMeta.Name].StorageClass,
-			pvols[item.ObjectMeta.Name].AttachementStatus,
-			pvols[item.ObjectMeta.Name].AccessMode,
-			pvols[item.ObjectMeta.Name].Node}})
+		if val, ok := pvols[item.ObjectMeta.Name]; ok {
```

Comment:
(optional) assign the values from val.StorageClass, val.AttachmentStatus to a variable scoped outside this if-else and then call `append` on the `rows`, this will reduce the lines one would have to read. The effect is minimal(barely 5-8lines less) so this comment is optional.

Reply:
- I will add variables.